docs/i18n/README.md

```
@@ -26,6 +26,12 @@ Narrative depth status: **enhanced bridge rollout completed for `zh-CN`/`ja`/`ru
   - `docs/i18n/el/`
 - Compatibility hubs at docs root (`docs/README.<locale>.md`) remain available during migration.
 
+## Structure
+
+- Docs structure map (language/part/function): [../structure/README.md](../structure/README.md)
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)